src-electron/db/query-config.js

```diff
@@ -211,15 +212,18 @@ async function insertOrUpdateAttributeState(
       attributeId,
       clusterRef
     )
-  if (
-    staticAttribute.storagePolicy ==
-    dbEnum.storagePolicy.attributeAccessInterface
-  ) {
-    staticAttribute.storagePolicy = dbEnum.storageOption.external
-  } else {
-    staticAttribute.storagePolicy = dbEnum.storageOption.ram
-  }
-
+  let forcedExternal = await queryUpgrade.getForcedExternalStorage(
+    db,
+    attributeId
+  )
+  let storageOption = await queryUpgrade.computeStorage(
+    db,
+    null,
```

Comment:
Why is this null instead of the cluster name?

Reply:
because clusterName is not needed when upgrading a new configuration so I used it as a differentiator

Reply:
never mind, you're right. will fix.